functions/src/import/index.ts

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2022 Meilisearch
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as admin from 'firebase-admin'
+import { CliConfig, parseConfig } from './config'
+import * as logs from '../logs'
+import { adaptDocument } from '../adapter'
+import { initMeilisearchIndex } from '../meilisearch/create-index'
+import { Index } from '../types'
+import { DocumentSnapshot } from 'firebase-functions/lib/providers/firestore'
```

Comment:
import should always be 
1. the external imports
2. the internal imports
```suggestion
import * as admin from 'firebase-admin'
import { DocumentSnapshot } from 'firebase-functions/lib/providers/firestore'
import { CliConfig, parseConfig } from './config'
import * as logs from '../logs'
import { adaptDocument } from '../adapter'
import { initMeilisearchIndex } from '../meilisearch/create-index'
import { Index } from '../types'

```